### PR TITLE
[Improvement] state/dynamodb - Enable AWS Backup

### PR DESCRIPTION
**Pull request by michaelwittig**
WIP

### CODE REVIEW

**Comment by andreaswittig** [comment]
@michaelwittig Is this PR ready for review?

**Comment by michaelwittig** [comment]
@andreaswittig yes, you can start with the review. One open question is if a DeletionPolicy is needed or not.

File: state/dynamodb.yaml [hunk 1]
```diff
@@ -309,6 +325,60 @@ Resources:
       - {'Fn::ImportValue': !Sub '${ParentAlertStack}-TopicARN'}
       OKActions:
       - {'Fn::ImportValue': !Sub '${ParentAlertStack}-TopicARN'}
+  BackupVault:
+    Type: 'AWS::Backup::BackupVault'
+    Properties:
+      # AccessPolicy: # FIXME later
+      BackupVaultName: !Ref 'AWS::StackName'
+      # EncryptionKeyArn: # FIXME later
```
Comment (andreaswittig) [inline state/dynamodb.yaml]: @michaelwittig What is missing here?

File: state/dynamodb.yaml [hunk 2]
```diff
@@ -309,6 +325,60 @@ Resources:
       - {'Fn::ImportValue': !Sub '${ParentAlertStack}-TopicARN'}
       OKActions:
       - {'Fn::ImportValue': !Sub '${ParentAlertStack}-TopicARN'}
+  BackupVault:
+    Type: 'AWS::Backup::BackupVault'
+    Properties:
+      # AccessPolicy: # FIXME later
```
Comment (andreaswittig) [inline state/dynamodb.yaml]: @michaelwittig Do we need a resource policy?

Comment (michaelwittig) [reply state/dynamodb.yaml]: no

File: state/dynamodb.yaml (hunk 1 above)
Comment (michaelwittig) [reply state/dynamodb.yaml]: I use the default key now to avoid the issue of loosing backups if the kms key is deleted.

Comment (andreaswittig) [reply state/dynamodb.yaml]: 👍

**Pull request merged**